src/ucm/cuda/cudamem.c

```diff
@@ -178,11 +180,11 @@ UCM_CUDA_ALLOC_FUNC(cuMemAlloc_v2, CUresult, CUDA_SUCCESS, arg0, CUdeviceptr, *,
 UCM_CUDA_ALLOC_FUNC(cuMemAllocManaged, CUresult, CUDA_SUCCESS, arg0,
                     CUdeviceptr, *, "size=%zu flags=0x%x", size_t, unsigned)
 UCM_CUDA_ALLOC_FUNC(cuMemAllocPitch, CUresult, CUDA_SUCCESS,
-                    ((size_t)arg1) * (arg2), CUdeviceptr, *,
+                    ((size_t)(*arg0)) * (arg2), CUdeviceptr, *,
```

Comment:
why changed? according to cuda doc:
https://docs.nvidia.com/cuda/cuda-driver-api/group__CUDA__MEM.html#group__CUDA__MEM_1gcbe9b033f6c4de80f63cc6e58ed9a45a
"Allocates at least WidthInBytes * Height bytes of linear memory on the device and returns in *dptr a pointer to the allocated memory. "
The allocation size should use WidthInBytes (and not *pPitch)